CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Here is a template for new release sections
 - Add the new parameter `scenario_description` to input files and docs and a section in autoreport describing the scenario being simulated (#722)
 - Add a new sub-section in RTD describing the suite of post-simulation tests carried out through functions in E4_verification.py (#753)
 - Add KPI individual sectors to the report (#757)
+- Added tests in tests_D2_model_constraints.py for the function adding the minimal renewable share constraint in D2_model_constraints.py (#742)
```

Comment:
Please name the function here and also add the test function name to the docstring of the function.

Reply:
Done